setup.py

```diff
@@ -33,7 +33,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='ora2',
-    version='1.1.10',
+    version='1.1.11',
```

Comment:
Please leave this at the current version for this PR, and I'll take care of updating it with translations as part of the [ORA release process](https://openedx.atlassian.net/wiki/display/ENG/ORA2+Release+Process). @cahrens, do you or your team have any work you'd like to release soon, or should I get this process started once this PR lands?